Update to core20 and gnome-3-38

hellsworth opened the pull request:
## Description

This change updates the base to core20 and the snapcraft extension to gnome-3-38.

Fixes # (issue)

## Type of change

Please check only the options that are relevant.

- [x] General maintenance
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

**Test Configuration**:

* OS (please include version):
* Any other relevant environment information:

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

Commit:
Update to core20 and gnome-3-38

kenvandine commented:
There is a commented out copy line that I think we'll still need, but probably doesn't work as it was.

Commit:
Update path to desktop icon file

Commit:
Update url to launchpad snap

hellsworth commented:
Fixed the path to the icon and looks fine. Also updated the readme to point to the correct url to the snap on launchpad, and added a manual workflow action. Should be good to merge now.

I tested a local build of this on my impish system, both x11 and wayland.

kenvandine reviewed (approved):
Looks good
